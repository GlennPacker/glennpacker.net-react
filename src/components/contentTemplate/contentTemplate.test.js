import React from 'react';
import { shallow } from 'enzyme';

import ContentTemplate from './index';

describe('Server Problems', () => {
    let wrapper;
    let content;
    let articles;

    beforeAll(() => {
        content = (<div>hello</div>)
        articles = [
            { a: 1 },
            { a: 2 }
        ]

        wrapper = shallow(
            <ContentTemplate
                content={content}
                articles={articles}
            />
        );
    })

    it('should render matching snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})