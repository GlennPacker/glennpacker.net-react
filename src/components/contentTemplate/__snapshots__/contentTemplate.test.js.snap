// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Server Problems should render matching snapshot 1`] = `
<Fragment>
  <div
    className="col-md-8 pb-2"
  >
    <div>
      hello
    </div>
  </div>
  <ArticleRow
    0={
      Object {
        "a": 1,
      }
    }
    1={
      Object {
        "a": 2,
      }
    }
  />
</Fragment>
`;
